pingen_env/views/base_config_views.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+   <record model="ir.ui.view" id="res_config_settings_view_form">
+    <field name="model">res.config.settings</field>
+    <field name="inherit_id" ref="pingen.res_config_settings_view_form" />
+    <field name="arch" type="xml">
+      <field name="pingen_clientid" position="attributes">
+        <attribute name="attrs" eval="False" />
```

Comment:
Are these overrides needed? I don't see any `attrs` defined in https://github.com/OCA/report-print-send/pull/315/files#diff-cdd40590b017ab65f0a06265805b28d4a3cf08b24966d0f5211a6167441758f6R17